src/components/LiveFeedItem.jsx

```diff
@@ -46,6 +45,15 @@ const Container = styled('div')(({ type, highlight }) => ({
       : type === 'myalert' && highlight !== 'true'
       ? '#ff59e8'
       : 'rgb(0, 30, 60)',
+
+  color:
```

Comment:
nit, there must be a better way to write this in. I'd say even a variable which you set with if will be easier to understand than nested ternary